### ISSUE TITLE
V3.2.2 using Bootstrap 5 alpha release components which are yet to stabilize

### ISSUE THREAD

**Issue by coolsam726** [coreui/coreui#143]
I noticed that the js files of the latest release (v3.2.2) are using Bootstrap 5 alpha! If this is the case then this is a major inconsistency especially for those of us using coreui in production because personally I am not ready to upgrade to Bootstrap 5 until it is stable. What's worse is that this change is in a minor release. I think if you intend to move to bootstrap 5 components, since that is a huge undertaking, it should be implemented in a new major release (v4.x).

**Comment by mrholek** [comment]
Bootstrap-based components without jQuery are available since the last year and we use them as the base of our own components since CoreUI 3.0.0 is available. If you compare our releases you will see that probably the number of Bootstrap was changed from v4.x.x to v5 and there are no significant changes in code.

As I mentioned before CoreUI based on Bootstrap and our library is 100% compatible with the latest stable version of Bootstrap, but Bootstrap development is really slow and we can wait for them, so we made a lot of improvements in their SCSS and JS files, for example, our components use Popper.js 2 when Boostrap still use Popper.js 1, we consider to move to TypeScript in CoreUI 4 while Bootstrap 5 use JavaScript.

To sum up, we use modified components from version 5 but, everything was checked by our developers, we made significant improvements, and everything is stable and production-ready.

**Comment by manuelmeurer** [comment]
@mrholek Is there a specific reason you didn't bump the major version to v4 when changing such a substantial dependency? I agree with @coolsam726, it's really unexpected so see such a change in a patch release.

Won't this break apps that upgrade to CoreUI 3.x and are using Bootstrap v4 specific features that were removed in Bootstrap v5?

**Comment by mrholek** [comment]
@manuelmeurer as I said before, we didn't change anything, we use Bootstrap based components without jQuery since CoreUI 3 was released.

**Comment by manuelmeurer** [comment]
Ok, maybe I misunderstood... when exactly did CoreUI start using components from Bootstrap v5 alpha instead of Bootstrap v4? Was ist from CoreUI v3 on?

Just to clarify, I'm not trying to criticize, I think CoreUI is great and I appreciate all the work that you put into it! 🙇

**Comment by mrholek** [comment]
Yes, it was from CoreUI v3 on, but please note that we made some changes, improvements, and tested everything during the alpha and beta phase. Please note that in future releases of version 3 you will see some features which will be available in Bootstrap 5 ex. grid with XXL breakpoint. We're a professional team focused only on CoreUI, that's why we have more components, styles, and we're ahead of Bootstrap and have more components.
